src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -220,6 +220,47 @@ To see this, take a look at the job script for one of the downstream tasks:
    configuration.
 
 
+.. _Sequential Trigger Spawning:
+
+Sequential Trigger Spawning
+---------------------------
+
+Parent-less tasks (having no graph dependencies on other tasks) naturally spawn
+out to the runahead limit. The consequence for workflows with large runahead
+limits and/or task count may not only be UI clutter but also needless trigger
+checking. An example of this would be ``wall_clock``, where there's clearly
+no point in checking the future if the present trigger is unsatisfied.
+
+Sequential trigger spawning is a way of altering the default spawning behavior
+of these parent-less tasks. This is achieved by setting any number of triggers
+as sequential so that associated tasks only spawn their next (cycle point)
+instance on satisfaction of these triggers.
+
+A trigger can be set as sequential in any/all of the following ways.
```

Comment:
```suggestion
A trigger can be set as sequential in the following ways.
```

Reply:
I guess I want to suggest you can use all with precedence taken into account.

Reply:
OK I don't mind if you keep that.  Maybe say "any or all of" though.